src/platform/plugins/shared/dashboard/public/dashboard_api/layout_manager/layout_manager.ts

```diff
@@ -42,29 +42,37 @@ import {
 } from '@kbn/presentation-publishing';
 import { asyncForEach } from '@kbn/std';
 
+import { DEFAULT_CONTROL_GROW, DEFAULT_CONTROL_WIDTH } from '@kbn/controls-constants';
+import type { PinnedControlLayoutState } from '@kbn/controls-schemas';
 import type { PanelResizeSettings } from '@kbn/presentation-util-plugin/public';
 import { PanelPlacementStrategy } from '@kbn/presentation-util-plugin/public';
 import type { DashboardState } from '../../../common';
 import { DEFAULT_PANEL_HEIGHT, DEFAULT_PANEL_WIDTH } from '../../../common/constants';
 import type { DashboardPanel } from '../../../server';
 import { dashboardClonePanelActionStrings } from '../../dashboard_actions/_dashboard_actions_strings';
 import { getPanelAddedSuccessString } from '../../dashboard_app/_dashboard_app_strings';
+import { getPanelSettings } from '../../panel_placement/get_panel_placement_settings';
+import { placeClonePanel } from '../../panel_placement/place_clone_panel_strategy';
+import { runPanelPlacementStrategy } from '../../panel_placement/place_new_panel_strategies';
 import { coreServices, usageCollectionService } from '../../services/kibana_services';
 import { DASHBOARD_UI_METRIC_ID } from '../../utils/telemetry_constants';
 import type { initializeTrackPanel } from '../track_panel';
 import { areLayoutsEqual } from './are_layouts_equal';
 import { deserializeLayout } from './deserialize_layout';
 import { serializeLayout } from './serialize_layout';
-import type { DashboardChildren, DashboardLayout, DashboardLayoutPanel } from './types';
 import {
-  getPanelSettings,
-  placeClonePanel,
-  runPanelPlacementStrategy,
-} from '../../panel_placement';
```

Comment:
super nit, but not sure why this was changed to deep imports since the functions are exported from the barrel file?

Reply:
Bad merge, fixing